V3_MIGRATION.md

```diff
@@ -220,10 +220,10 @@ xrplClient.submit(signedTransaction);
 To create a `MultiSignedTransaction`, use `BcSignatureService` like this:
 
 ```java
-Set<SignatureWithPublicKey> signers = Lists.newArrayList(aliceKeyPair, bobKeyPair).stream()
+Set<Signer> signers = Lists.newArrayList(aliceKeyPair, bobKeyPair).stream()
   .map(keyPair -> {
       Signature signedPayment = signatureService.multiSign(keyPair.privateKey(), unsignedPayment);
-      return SignatureWithPublicKey.builder()
+      return Signer.builder()
```

Comment:
What do you think about returning `Signer` from `SignatureService.multiSign`?

Reply:
We decided that returning `Signer` from `multiSign` would require the implementation of `multiSign` to call `this.derivePublicKey`. In most cases, the public key will already exist in memory at the time of calling `multiSign`, so returning a `Signer` would introduce an extraneous calculation or call to an external system. This seems like it could create performance problems for developers, so we chose to keep this as `Signature`